cicecore/drivers/standalone/cice/CICE_RunMod.F90

```diff
@@ -318,6 +383,9 @@ subroutine ice_step
             enddo
          endif
 
+#ifdef OMP_TIMERS
+         call ice_barrier()
+#endif
```

Comment:
I'm curous why we need a barrier at a few places ? this is an MPI barrier, no? How does it interact with OpenMP ?

Reply:
Same.